### PR TITLE
[simd] Add new port

### PR DESCRIPTION
**Pull request by Edi61**
**Describe the pull request**
Created new port for https://github.com/ermig1979/Simd
related also to issue https://github.com/microsoft/vcpkg/issues/15008

- #### What does your PR fix?
No fix, introduction of a new port

- #### Which triplets are supported/not supported? Have you updated the [CI baseline]
Windows, Linux. Yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes, maintenace guide is checked e.g. replaced vcpkg_build_msbuild by vcpkg_install_msbuild

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Comment by erdogan61** [comment]
@microsoft-github-policy-service agree

**Review by Adela0814 — CHANGES_REQUESTED** [review]

File: scripts/ci.baseline.txt [hunk 1]
```diff
@@ -987,6 +987,11 @@ shogun:x86-windows        = skip
 signalrclient:x64-uwp=fail
 signalrclient:arm-uwp=fail
 simbody:arm64-windows=fail
+simd:arm-uwp=fail
+simd:arm64-windows=fail
+simd:x64-uwp=fail
+simd:x64-linux=fail
```
[the remark this answers is not in the document]
Comment (erdogan61) [reply scripts/ci.baseline.txt:993]: I don't have linux and also not have so much experience about cmake, therefore I have let it open for now. But maybe you can give me some idea how I can test it on a windows system?

[the remark this answers is not in the document]
Comment (erdogan61) [reply scripts/ci.baseline.txt:993]: I will try with Ununtu Wsl and provide a solution.

File: scripts/ci.baseline.txt [hunk 2]
```diff
@@ -987,6 +987,10 @@ shogun:x86-windows        = skip
 signalrclient:x64-uwp=fail
 signalrclient:arm-uwp=fail
 simbody:arm64-windows=fail
+simd:arm-uwp=fail
+simd:arm64-windows=fail
+simd:x64-uwp=fail
+simd:x64-osx=fail
```
Comment (Adela0814) [inline scripts/ci.baseline.txt:993]: Does `simd` really not support these triplets? I know you have no corresponding machine. Can you remove this and run again on CI?

**Review by Adela0814 — CHANGES_REQUESTED** [review]

File: scripts/ci.baseline.txt (hunk 2 above)
Comment (erdogan61) [reply scripts/ci.baseline.txt:993]: ->updated supports field in simd.json.

Comment (Adela0814) [reply scripts/ci.baseline.txt:993]: You should remove the unsupported from baseline.

**Review by Adela0814 — CHANGES_REQUESTED** [review]

**Comment by erdogan61** [comment]
Removed all mentioned builds from Ci-Baseline, but uwb and osx failed since not supported. Arm seem to be supported: I have tried some update,
But i cannot access the log files from Azure, is there some possibility?

**Comment by erdogan61** [comment]
To sum up:
- uwb and osx are not supported
- arm* is supported generally (therefore added the support), but arm64 with MSVC compiler is not (-> CI build will fail)
Updated also supports field in vcpkg.json

File: scripts/ci.baseline.txt [hunk 3]
```diff
@@ -987,6 +987,10 @@ shogun:x86-windows        = skip
 signalrclient:x64-uwp=fail
 signalrclient:arm-uwp=fail
 simbody:arm64-windows=fail
+simd:arm-uwp=fail
+simd:x64-uwp=fail
+simd:x64-osx=fail
+simd:arm64-windows=fail
```
Comment (Adela0814) [inline scripts/ci.baseline.txt:993]: This is unnecessary. You can delete it because it is added to the supports.

**Review by Adela0814 — CHANGES_REQUESTED** [review]

**Review by Adela0814 — DISMISSED** [review]

**Pull request merged**